packages/itwinui-css/src/bridge.scss

```diff
@@ -195,21 +195,21 @@
 
   .iui-checkbox-wrapper {
     &:not(:where([data-iui-status])):not([data-iui-disabled='true']) {
-      // Kiwi labels are slightly muted
-      color: var(--kiwi-color-text-neutral-secondary);
+      // v5 labels are slightly muted
+      color: var(--ids-color-text-neutral-secondary);
     }
   }
 
   /* ---------------------------------------------------------------------------- */
   /* DropdownMenu */
 
   .iui-menu {
-    // Kiwi uses different surface colors for the dropdowns
-    background-color: var(--kiwi-color-bg-surface-tertiary);
+    // v5 uses different surface colors for the dropdowns
+    background-color: var(--ids-color-bg-elevation-elevation-level-1);
```

Comment:
This should be `bg-elevation-level-1`. There was a typo in my description of the breaking change.

@r100-stack Can you open another PR to fix it?